### PR TITLE
text-decoration implemented

### PR DESCRIPTION
**Pull request by enchev**
Implemented #875

### CODE REVIEW

File: ui/styling/converters.ts [hunk 1]
```diff
@@ -24,6 +24,19 @@ export function textAlignConverter(value: string): string {
     }
 }
 
+export function textDecorationConverter(value: string): string {
+    switch (value) {
+        case enums.TextDecoration.none:
+        case enums.TextDecoration.underline:
+        case enums.TextDecoration.lineThrough:
+            return value;
+            break;
```
Comment (hshristov) [inline ui/styling/converters.ts]: no need for break here.

File: ui/styling/style.ts [hunk 2]
```diff
@@ -536,6 +540,19 @@ export class Style extends DependencyObservable implements styling.Style {
...
 
+    get textDecoration(): string {
+        return this._getValue(textDecorationProperty);
+    }
+    set textDecoration(value: string) {
+        this._setValue(textDecorationProperty, value);
+    }
+
+    public _updateTextDecoration() {
+        if (this._getValue(textDecorationProperty) !== enums.TextDecoration.none) {
```
Comment (hshristov) [inline ui/styling/style.ts]: What will happen if I set textDecoration to Underline then to None again?

File: ui/styling/stylers.android.ts [hunk 3]
```diff
@@ -448,6 +461,26 @@ export class TextViewStyler implements definition.stylers.Styler {
...
             TextViewStyler.resetTextAlignmentProperty,
             TextViewStyler.getNativeTextAlignmentValue), "Button");
+
+        style.registerHandler(style.textDecorationProperty, new stylersCommon.StylePropertyChangedHandler(
+            TextViewStyler.setTextDecorationProperty,
+            TextViewStyler.resetTextDecorationProperty), "Button");
+    }
+}
+
+function setTextDecoration(view: android.widget.TextView, value: string) {
```
Comment (hshristov) [inline ui/styling/stylers.android.ts]: Can we set both of them at the same time? (e.g. Underline and Strike_thru)

Comment (enchev) [inline ui/styling/stylers.android.ts]: No.

**Comment by enchev** [comment]
run ci

**Comment by ErjanGavalji** [comment]
:+1:

**Comment by enchev** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Pull request merged**